File: week 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get size of grid
    int n;
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 15);  // Ensure n is between 1 and 9 inclusive

    // Print grid of bricks
    for (int i = 0; i < n; i++)
    {
        // Print leading spaces
        for (int f = 0; f < n - i - 1; f++)
        {
            printf("");
        }

        // Print left pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        // Print gap
        printf("  ");

        // Print right pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        // Move to next row
        printf("\n");
    }
}

File: week 6/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if 9 > n > 0:
        break

for i in range(n):
    print(" " * (n - i - 1), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1), end="")
    print()
